src/language_server.ts

```diff
@@ -13,10 +15,13 @@ import { fileOrDirectoryExists, log } from './utils';
 import { v4 as uuidv4 } from 'uuid';
 import { getLanguageMap } from './extension';
 
-enum LanguageServerDiagnoseResult {
+const REQUIREMENTS_PATH = path.join(__dirname, "..", "requirements.txt");
```

Comment:
Do you know if this is the same in packaged form?

Reply:
I tested it by installing the .vsix produced by jenkins and it worked for me